<?xml version="1.0"?>
<launch>
	<node name="myPublisher" pkg="load_uwbdata" type="myPublisher" output="screen">

		<param name="node0id" value="101" />
		<param name="node1id" value="102" />
		<param name="node2id" value="105" />
		<param name="node3id" value="106" />

		<param name="device" value="/dev/ttyACM0" />
		<param name="port" value="usb" />
	
		<param name="node0_posx" value="-3.000" />
		<param name="node0_posy" value="-3.000" />
		<param name="node0_posz" value="-1.710" />
	
		<param name="node1_posx" value="3.000" />
		<param name="node1_posy" value="-3.000" />
		<param name="node1_posz" value="-1.210" />

		<param name="node2_posx" value="3.000" />
		<param name="node2_posy" value="3.000" />
		<param name="node2_posz" value="-1.310" />

		<param name="node3_posx" value="-3.000" />
		<param name="node3_posy" value="3.000" />
		<param name="node3_posz" value="-1.310" />
		
	</node>

	<node name="datarecord" pkg="rosbag" type="record" 
		args="-o $(find load_uwbdata)/bag/dis /myPublisher/uwb_distance /viconXbee_node/mocap/pose">
	</node>
	
	<include file="$(find viconXbee)/launch/viconXbee.launch"/>

</launch>
